distribution/ddf-common/src/main/resources/etc/users.properties

```diff
@@ -32,5 +32,5 @@
 
 # DDF Changes: changed username/pw to admin/admin, added localhost certificate user
 
-admin=admin,group,admin,manager,viewer,webconsole
-localhost=localhost,group,admin,manager,viewer,webconsole
\ No newline at end of file
+admin=admin,group,admin,manager,viewer,webconsole,system-admin
```

Comment:
There are two "admin"s (admin=**admin**,group,**admin**,...) on the list... should one be removed since we're here making changes anyway? Or is that second admin necessary somehow?